tests/snapshots/test_cli__markdown-help.snap

```diff
@@ -304,20 +371,118 @@ Print help information
 This subcommand accepts all the [global options](#global-options)
 
 <br><br><br>
-## cargo vet add-unaudited
-Mark `$package $version` as unaudited
+## cargo vet regenerate
+Explicitly regenerate various pieces of information
+
+There are several things that `cargo vet` *can* do for you automatically but we choose to make
+manual just to keep a human in the loop of those decisions. Some of these might one day become
+automatic if we agree they're boring/reliable enough.
+
+See the subcommands for specifics.
+
+### USAGE
+```
+cargo vet regenerate [OPTIONS] <SUBCOMMAND>
+```
+
+### OPTIONS
+#### `-h, --help`
+Print help information
+
+### GLOBAL OPTIONS
+This subcommand accepts all the [global options](#global-options)
+### SUBCOMMANDS
+* [exemptions](#cargo-vet-exemptions): 
+* [imports](#cargo-vet-imports): Suggest some low-hanging fruit to review
+* [audit-as-crates-io](#cargo-vet-audit-as-crates-io): Initialize cargo-vet for your project
+* [help](#cargo-vet-help): Print this message or the help of the given subcommand(s)
```

Comment:
this is a usable result, but we should probably properly namespace the anchors...

Reply:
Yeah, that would probably be a good idea. Perhaps change the todo list to include a `(String, &mut clap::Command)` instead, so that you can pass the "full name" of the command alongside it?